tests/functional/fixtures/server_with_batching_fixtures.py

```diff
@@ -91,3 +143,29 @@ def start_server_batch_model_bs4(request, get_image, get_test_dir,
     assert running is True, "docker container was not started successfully"
 
     return container
+
+
+@pytest.fixture(scope="class")
+def start_server_batch_model_auto_bs4_2out(request, get_image, get_test_dir,
```

Comment:
We set batch size to 4 in this fixture. "auto" in name might be a little confusing.